watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1690/tests/test_amo_1690_sweep.py

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import os
+import pytest
+import tempfile
+from watertap.examples.flowsheets.case_studies.wastewater_resource_recovery.amo_1690.amo_1690_sweep import (
+    run_analysis,
+)
+
+sweep_list = [
+    1,
+    2,
+    3,
+]
```

Comment:
DO we need all 3 for the test?

Reply:
I suppose we don't "need' to test all three cases, but I think it's probably best practice to do so. The tests use nx=2, so it shouldn't take much additional time to test them all.